tcms/telemetry/static/telemetry/js/testing/test-case-health.js

```diff
@@ -95,3 +99,19 @@ function renderFailedExecutionsColumn(data) {
 function renderPercentColumn(data) {
     return Number.parseFloat(data.count.fail / data.count.all * 100).toFixed(1);
 }
+
+function renderVisualPercent(data) {
+    const failPercent = data.count.fail / data.count.all * 100;
+
+    const colors = [];
+    const step = 20;
+    for (i = 0; i < 5; i++) {
+        if(failPercent > i * step) {
+            colors.push('red')
+        } else {
+            colors.push('green')
+        }
+    }
+
+    return colors.reduce((prev, color) => prev += `<span style='width: 15%; height: 10px; display: inline-block; padding-left: 1px; background-color: ${color}'></span>\n`, '')
```

Comment:
Add a class for this span element and move everything except color in CSS definitions.

Also the current size looks to me a bit big. I tried 7x7 px and looks a bit better (font size appears to be 12px here).